common/djangoapps/student/management/commands/anonymousid_remap.py

```diff
@@ -33,9 +36,22 @@ def handle(self, *args, **options):
                         student.user.id
                     )
                 )
+                old_id = student.anonymous_user_id
                 student.anonymous_user_id = digest
                 student.save()
-                remaped_count += 1
+                remaped.append({
+                    "old_id": old_id,
+                    "new_id": digest,
+                    "email": student.user.email,
+                    "course_id": student.course_id.to_deprecated_string()
+                        .encode('utf-8') if student.course_id else ""
+                })
+        filename_postfix = datetime.datetime.now(tzlocal())
+        filename = "anonymous_ids_{}.json".format(
```

Comment:
here is generated filename example 
```
# date
Thu Mar 17 12:00:06 UTC 2016
```
`anonymous_ids_2016-03-17_12:00:14_EDT.json`